cdk-erigon-sequencer-params.yal

```diff
@@ -0,0 +1,181 @@
+---
+# Deployment parameters.
+# The deployment process is divided into various stages.
+
+# Deploy local L1.
+deploy_l1: true
+
+# Deploy zkevm contracts on L1 (and also fund accounts).
+deploy_zkevm_contracts_on_l1: true
+
+# Deploy zkevm node and cdk peripheral databases.
+deploy_databases: true
```

Comment:
when convenient, interested to know what comprises the full set of dbs here ? devTools is working on longer-running devnets deployed in GKE and we're actively mapping what the persistent storage will need to look like (in totality)

Reply:
full set of db is represented on this diagram: https://github.com/0xPolygon/kurtosis-cdk/blob/main/docs/img/architecture.png